src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+@import "../utils/variables.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $blockGap ;
+  width: $blockSize*7 + $blockGap*6 + 20px;
+  padding: 10px;
+
+  &__block {
+    box-sizing: border-box;
+    width: $blockSize;
+    height: $blockSize;
+    background-color: #eee;
+    border: 1px solid black;
+    transition: transform 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      cursor: pointer;
+      background-color: pink;
+    }
+    
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        display: block;
+        text-align: center;
+        width: 100%;
+        height: 100%;
+        line-height: $blockSize;
+        content: "#{$i}";
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+  }
+}
+
+@each $day, $value in $weekDay {
+  .calendar--start-day--#{$day} {
+    .calendar__block:first-child {
```

Comment:
Why don’t you use nested? `&`

Reply:
I was trying to write with &, but it`s stops working